nifi-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -60,16 +60,10 @@ dataflows.
 
 WARNING: Before proceeding, check the Admin Guide to confirm you have the <<administration-guide.adoc#system_requirements,minimum system requirements>> to install and run NiFi.
 
-NiFi can be downloaded from the link:http://nifi.apache.org/download.html[NiFi Downloads page^]. There are two packaging options
-available:
+NiFi can be downloaded from the link:http://nifi.apache.org/download.html[NiFi Downloads page^]. After downloading the version of NiFi
+that you would like to use, simply extract the zip archive to the location that you wish to run the application from.
 
-- a "tarball" (tar.gz) that is tailored more to Linux
-- a zip file that is more applicable for Windows users
-
-macOS users may also use the tarball or can install via link:https://brew.sh[Homebrew^] by simply running the command `brew install nifi` from the command line terminal.
-
-For users who are not running macOS or do not have Homebrew installed, after downloading the version of NiFi that you
-would like to use, simply extract the archive to the location that you wish to run the application from.
+macOS users may also install via link:https://brew.sh[Homebrew^] by simply running the command `brew install nifi` from the command line terminal.
```

Comment:
Since the `brew.sh` approach is not supported by the Apache NiFi project itself, this is a good opportunity to remove this reference.
```suggestion
```

Reply:
Done.  I see Apache NiFi project members are not normally the ones to update the nifi Homebrew formula.